pkg/virt-config/virt-config.go

```diff
@@ -487,6 +485,5 @@ func (c *ClusterConfig) GetInstancetypeReferencePolicy() v1.InstancetypeReferenc
 }
 
 func (c *ClusterConfig) ClusterProfilerEnabled() bool {
-	return c.GetConfig().DeveloperConfiguration.ClusterProfiler ||
-		c.isFeatureGateDefined(featuregate.ClusterProfiler)
```

Comment:
Please, add a note about this change in the commit message, specifying that this FG is already GA. Thanks